script.js

```diff
@@ -0,0 +1,82 @@
+//turn
+
+var turn = 0;
+var squares = $("td");
+
+addLetter = function() {
+
+  //if turn = even
+  if (turn % 2 === 0) {
+    //click event should append X to appropriate square
+    console.log(this);
+    $(this).text("X");
+    winner();
+  }
+  //if turn = odd
+  else {
+    //click event should append O to appropriate square
+    console.log(this);
+    $(this).text("O");
+    winner();
+  }
+  turn += 1;   //Even though this action is in the function scope, it is updating the value of a global variable.  So even thought turn = 0 in global scope, after you click, it refers to the global variable and updates it by one.
+};
```

Comment:
what happens if i click the same block twice? Something to consider when you want to polish this up. Maybe in the `winner()` function you can turn off an event listener or add that feature to the `addLetter` function itself.